packages/mobile/src/harmony-native/components/input/TextInput/TextInput.tsx

```diff
@@ -244,7 +244,7 @@ export const TextInput = forwardRef(
             <InsetShadow
               containerStyle={css({
                 width: '100%',
-                height: isSmall ? 34 : 64,
+                height: isSmall ? 48 : 64,
```

Comment:
do we want this to apply to all other small text-input, im thinking filter input in library?

Reply:
pinged design to confirm this